gossip/src/duplicate_shred.rs

```diff
@@ -35,10 +36,10 @@ pub struct DuplicateShred {
     shred_index: u32,
     shred_type: ShredType,
     // Serialized DuplicateSlotProof split into chunks.
-    num_chunks: u8,
-    chunk_index: u8,
+    pub(crate) num_chunks: u8,
+    pub(crate) chunk_index: u8,
     #[serde(with = "serde_bytes")]
-    chunk: Vec<u8>,
+    pub(crate) chunk: Vec<u8>,
```

Comment:
Rewrote the code to not use chunk. But I do need to check num_chunks and chunk_index so I can piece together the whole proof, there is no way around it.

Reply:
Ok, instead of `pub(crate)` fields, better only add accessor methods.
```rust
pub(crate) fn num_chunks(&self) -> u8 {
   self.num_chunks
}
```
similarly for `chunk_index`.

Reply:
done